opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -227,7 +227,19 @@ namespace Opm {
 
         using namespace std::placeholders;
         const auto handler = std::bind( handleKW, _1, schedule, props, n_xyz );
+
+#ifdef _MSC_VER
+        for (auto& x : section)
+        {
+            auto keywords = handler(x);
+            for (auto& keyword : keywords)
+            {
+                this->keywords.push_back(keyword);
+            }
+        }
+#else
         this->keywords = fun::concat( fun::map( handler, section ) );
```

Comment:
> The for loop is an abstraction over "two lines of code". Why doesn't that apply?

it does, that's my point: the "plain old stupid" way with the for-loop would not require you to know anything about `fun::` and `boost::bind` and it would not even be more lines  ;)

> Sure. But an old dog could learn new tricks.

true, but it's only necessary if the new trick is sufficiently better than the old way, and that's not the case here, IMO. (N.B: over-abstraction is a common problem I see through the OPM codebase: abstraction can be a good thing, but it is a double-edged sword and IMO a cure worse than the disease if there are only a very limited number of users for an API.)